src/mpcCoreKit.ts

```diff
@@ -1200,7 +1200,8 @@ export class Web3AuthMPCCoreKit implements ICoreKit {
     }
 
     if (tssPubKey.length === FIELD_ELEMENT_HEX_LEN + 1) {
-      tssPubKey = tssPubKey.subarray(1);
+      // Buffer is needed as react native buffer polyfill not able to polyfill subarray function
+      tssPubKey = Buffer.from(tssPubKey.subarray(1));
```

Comment:
Hm, I’m trying to understand what is the issue I’m more detail here. You state that subarray function is not available on react native polyfill, but you still use it here? How does wrapping with Buffer.from resolve the issue?